src/services/get-repos.ts

```diff
@@ -22,8 +22,8 @@ export async function getRepos (token: string, autoDiscoverRepos: AutoDiscoverRe
     auth: token
   })
 
-  const { data }: { data: repository[] } = await octokit
-    .request(`GET /${accountType}s/{${accountType}}/repos`, { [accountType]: owner })
+  const data = await octokit
+    .paginate(octokit.rest.repos.listForAuthenticatedUser, { owner, per_page: 100 })
```

Comment:
Thanks for contributing.
It is a very good point that I did not take into account, but what would happen if the account has more than 100 repositories?

Reply:
As per [Octokit JS SDK for Pagination](https://github.com/octokit/octokit.js#pagination),

> All REST API endpoints that paginate return the first 30 items by default. If you want to retrieve all items, you can use the pagination API. The pagination API expects the REST API route as first argument, but you can also pass any of the octokit.rest.*.list* methods for convenience and better code readability.

Thought this API call is not memory efficient, it will return all entries in a single method call.